restapi/src/main/java/io/stargate/web/resources/Converters.java

```diff
@@ -72,6 +74,14 @@
 public class Converters {
 
   private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+
+  static {
+    // For Java 8 date/time types we need:
+    OBJECT_MAPPER.registerModule(new JavaTimeModule());
+    // and by default date/time values written as numbers but we prefer ISO-8601:
+    OBJECT_MAPPER.disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS);
```

Comment:
But even with module, by default timestamp values are serialized as numbers; in this case an odd floating-point number. This is rarely what is wanted so change to use textual formatting, ISO-8601 value that is format being sent in the first place.